lib/middleware/blp-session.ts

```diff
@@ -20,6 +20,7 @@ function createSession(sessionId: string): Promise<blpapi.Session>
 {
     if (!(sessionId in SESSION_STORE)) {
         var s = new blpapi.Session(DEFAULT_SESSION_OPTIONS);
+        s.setMaxListeners(0);   // Remove max listener limit for blpSession
```

Comment:
Setting `EventEmitter.defaultMaxListeners` property, according to Node's doc, will affect all EventEmitter instances, not only limited to `blpapi.Session`.

Instead, I will move the `setMaxListener` in the `blpapi.Session` constructor so it will be called for all instance of it.